packages/sanity/src/core/i18n/fallback.ts

```diff
@@ -0,0 +1,64 @@
+import {memoize} from 'lodash'
+import {type Resource, createInstance} from 'i18next'
+import type {LocaleSource} from './types'
+import {studioLocaleNamespace} from './localeNamespaces'
+import {defaultLocale, usEnglishLocale} from './locales'
+import {isStaticResourceBundle} from './helpers'
+
+const shouldEscape = typeof window === 'undefined' || typeof document === 'undefined'
+const fallbackLocales: LocaleSource['locales'] = [
+  {id: defaultLocale.id, title: defaultLocale.title},
+]
+
+/**
+ * Returns a fallback source for internationalization that can be used in cases where you need
+ * access to the "framework" of translation, but do not have access to an actual source.
+ *
+ * Memoized - only initializes the i18n instance once, and maintains the identity of the source.
+ *
+ * ⚠️ NOTE: This will only have the base studio strings defined - no custom bundles or plugins.
+ *
+ * @returns The fallback source
```

Comment:
Do we want to link it to the type `LocaleSource`? 🤔

Reply:
Given this is internal, should be enough to follow the return type